src/mgr/DaemonServer.cc

```diff
@@ -724,3 +729,23 @@ bool DaemonServer::handle_command(MCommand *m)
     return true;
   }
 }
+
+void DaemonServer::send_report(bool)
```

Comment:
okay! we always assigned true to it before this change, so thought it was a placeholder.